queries/useProfileSocialsQuery.ts

```diff
@@ -96,7 +108,7 @@ export const fetchProfileSocialsQuery = (
   account: string,
   peerAddress: string
 ) => {
-  return queryClient.fetchQuery<ProfileSocialsData>(
+  return queryClient.fetchQuery<IProfileSocials | null>(
```

Comment:
Instead of this I love having something like in `useConversationMessages`

```

export type ConversationMessagesQueryData = Awaited<
  ReturnType<typeof conversationMessagesQueryFn>
>;
```

and reusing `ConversationMessagesQueryData` everywhere we use `queryClient...`